src/components/CrucibleFilter.vue

```diff
@@ -1,133 +1,132 @@
 <template>
-  <h1>This is a Filter</h1>
-  <div class="container">
-    <input id="All" type="radio" name="categories" value="All" checked />
-    <input id="CSS" type="radio" name="categories" value="CSS" />
-    <input id="JavaScript" type="radio" name="categories" value="JavaScript" />
-    <input id="jQuery" type="radio" name="categories" value="jQuery" />
-    <input id="WordPress" type="radio" name="categories" value="WordPress" />
-    <input id="Slider" type="radio" name="categories" value="Slider" />
-    <input
-      id="fullPage.js"
-      type="radio"
-      name="categories"
-      value="fullPage.js"
-    />
-
-    <div class="filters">
-      <div class="dropdown">
-        <label for="All">All</label>
-      </div>
-      <div class="dropdown">
-        <label for="CSS" @click="toggleDropdown">Course</label>
-        <ol v-if="showDropdown" class="dropdown-menu">
-          <li>Vet2011</li>
-          <li>Vet2012</li>
-        </ol>
-      </div>
-      <div>
-        <label for="JavaScript">Topic</label>
-        <ol>
-          <li>Physiology</li>
-        </ol>
-      </div>
-      <div>
-        <label for="jQuery">Animals</label>
-        <ol>
-          <li>Horse</li>
-        </ol>
-      </div>
+  <!-- <div class="crucible-filter-container"> -->
+  <div class="crucible-filters">
+    <div class="crucible-filter-dropdown">
+      <label for="All" @click="resetFilter">All</label>
+    </div>
+    <div
+      v-for="(items, key) in category"
+      :key="key"
+      class="crucible-filter-dropdown"
+    >
+      <label @click="toggleDropdown(key)">{{ key }}</label>
+      <ol v-show="showDropdown[key]" class="crucible-filter-dropdown-menu">
+        <li
+          v-for="(item, index) in items"
+          :key="index"
+          @click="getFilterTag(item)"
+        >
+          {{ item }}
+        </li>
+      </ol>
```

Comment:
I would also use `ul` rather than `ol` as we're not numbering anything in ranking for accessibility concerns.

Reply:
fixed